Solution1/Solution1.Tests/Integration/ConsoleAppTests.cs

```diff
@@ -33,5 +42,38 @@ public async Task Communicate_EnterCityName_HandlingExceptionAndShowNoticeAsync(
             var result = consoleOutput.ToString().Replace("Please, enter city name:\r\n", string.Empty);
             Assert.Matches(pattern, result);
         }
+
+        [Fact]
+        public async Task Main_GetForecastWeather_Seccess()
+        {
+            // Arrange
+            var cityName = "Minsk";
+            var countDays = 2;
+            var menu = Menu.GetMenuRepresentation();
```

Comment:
you use this method only in this class ConsoleAppTests. just make a private method

Reply:
+